content/zh/docs/tasks/debug-application-cluster/falco.md

```diff
@@ -0,0 +1,239 @@
+---
+reviewers:
+- soltysh
+- sttts
+- ericchiang
+content_template: templates/concept
+title: 使用Falco审计
```

Comment:
a general rule, leave one space between English and Chinese words ...
pls examine the whole page for similar issues